_vendors/globalprotect.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: GlobalProtect 
+ios_app: webview
```

Comment:
I can't post publicly (could share privately) but it is definitely possible to configure GlobalProtect in a way that supports FIDO2 on iOS. It pops the user all the way out to Safari and they perform the authentication flow there.

Reply:
We've seen this behavior inconsistently in the past week, but official PAN support says it is not possible.  We cannot replicate this 100%.  I would love if you can share how... The Portal setting for "Use default browser: yes" does not matter for mobile in our experience.

Reply:
+1 the portal setting for "Use default browser: yes" specifically does not apply to mobile (iOS and Android) in our experience as well - using Okta SAML. In our tests the iOS and Android apps use a webview browser - every once in a while, seemingly randomly, the iOS app will push auth to the default browser. But it's so inconsistent it cannot be relied upon.

Reply:
[This is the setting which seems to be providing the desired behavior.](https://docs.paloaltonetworks.com/globalprotect/10-0/globalprotect-admin/authentication/set-up-external-authentication/set-up-saml-authentication/enable-the-default-browser-for-saml-authenitcation)

My SME is indicating there was something wrong in [this earlier version of the documentation](https://docs.paloaltonetworks.com/globalprotect/5-2/globalprotect-app-new-features/new-features-released-in-gp-app/default-browser-for-saml-authentication) which took Palo Alto several months to correct. They also mentioned the outdated configuration being something the client downloads after it connects to VPN. In order to make client uses the default browser for the initial connection, there are config need to be done during installation.

Reply:
Thank you so much, this helped us solve that oddity.